### DISCUSSION TITLE
Strange performance

### DISCUSSION THREAD

**Discussion by wcurti** [vladmandic/sdnext#673] [General]
I'm having a strange performance issue and I'm wondering if anyone else has noticed it. Basically, for every generation, no matter what sampler I use and no matter what settings I use (e.g. SDP/doggettx's), it will generate two steps and then pause for nearly a second, then generate two more steps and pause for nearly a second.

It has slowed down overall generation time significantly.

**Comment by DyingWorld** [comment]
I'm seeing a similar problem since I upgraded to the latest build with the new requirements. Performance is 3x slower on my 3060.

**Comment by Nem-z** [comment]
I have this behavior too. Didn't have this issue prior to last commit. I reseted my `config.json` but the problem persist.
Here are some benchmarks where you can see I lost quite a lot of performance:
![image](https://user-images.githubusercontent.com/63961359/235500854-07a97029-4fc0-4bfe-bca5-2ba2e6bf26cb.png)

**Comment by vladmandic** [comment]
yup, known, recent code change upstream is the cause and i've notified the author:
<https://github.com/KohakuBlueleaf/a1111-sd-webui-lycoris/issues/21>

as a temporary workaround, you can disable the extension `a1111-sd-webui-lycoris`

**Reply by wcurti** [reply]
Okay, thank you! I was hoping it was something like this and not something on my end.

**Reply by DyingWorld** [reply]
Thanks for the quick reply! That fixed it for me

**Comment by elen07zz** [comment]
still not fixed right? my speeds are way slower.
